extensions/sql-database-projects/package.json

```diff
@@ -36,6 +36,41 @@
         "command": "sqlDatabaseProjects.open",
         "title": "%sqlDatabaseProjects.open%",
         "category": "%sqlDatabaseProjects.displayName%"
+      },
+      {
+        "command": "sqlDatabaseProjects.close",
```

Comment:
Make sure you're hiding these from the command palette for the ones that don't work from that context.